UlKit/UlKit/Feed/Cell/FeedPostsViewCell.swift

```diff
@@ -0,0 +1,285 @@
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let nickNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Constants.fontVerdanaBold, size: 12)
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let shareButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.tintColor = .black
+        button.setImage(.shareIcon, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let postImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let heartButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.tintColor = .black
+        button.setImage(.heartIcon, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let commentButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.tintColor = .black
+        button.setImage(.commentIcon, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let sendButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.tintColor = .black
+        button.setImage(.sendIcon, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let bookmarkButton: UIButton = {
+        let button = UIButton(type: .system)
+        button.tintColor = .black
+        button.setImage(.bookmarkIcon, for: .normal)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    private let likesLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Constants.fontVerdanaBold, size: 10)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let commentLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: Constants.fontVerdana, size: 10)
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private let myImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = .girl
+        imageView.contentMode = .scaleAspectFit
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let commenTextLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Комментировать..."
```

Comment:
```suggestion
        label.text = ""
```
Текстовки тоже в константы